templates/CRM/Event/Form/ManageEvent/Location.hlp

```diff
@@ -0,0 +1,24 @@
+{*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+*}
+{htxt id="id-is_show_location-title"}
+  {ts}Show Location{/ts}
+{/htxt}
+{htxt id="id-is_show_location"}
+  {ts}If this option is selected, the location will be shown in various places including:
+    <ul>
+      <li>the Event Information page</li>
+      <li>event registration confirmation emails</li>
+      <li>the HTML, RSS and ICALENDAR feeds of event information</li>
+    </ul>
```

Comment:
@aydun 
Can we remove the in first and last and capitalise all or none

- Event Information page
- Event registration confirmation emails
- HTML, RSS and ICALENDAR feeds of event information